client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -57,21 +57,15 @@ export class CommentDetailReply extends Component {
 	};
 
 	getTextareaPlaceholder = () => {
-		const { authorDisplayName, comment: { status }, translate } = this.props;
-
-		if ( 'approved' !== status ) {
```

Comment:
removing this `if` and keeping the last return should be enough:

```js
getTextareaPlaceholder = () => {
	const { authorDisplayName, translate } = this.props;

	return authorDisplayName
		? translate( 'Reply to %(commentAuthor)s…', {
				args: { commentAuthor: authorDisplayName },
			} )
		: translate( 'Reply to comment…' );
};
```

Reply:
I know, it's just that I'm bugged by how Prettier indents this kind of ternaries, breaking ESLint (on a rule not included in the pre-commit hooks, to be fair, but it still shows up as error in my editor).

Reply:
I know what you mean @Copons (dang squigglies!), but dropping the `if` does make a nicer diff.